tests/test_ai.py

```diff
@@ -71,3 +77,18 @@ def test_milvus_health(container_per_test):
     )
     container_per_test.connection.check_output("etcd --version")
     container_per_test.connection.check_output("milvus")
+
+
+@pytest.mark.parametrize(
+    "container_per_test",
+    [PYTORCH_CONTAINER],
+    indirect=["container_per_test"],
+)
+def test_pytorch_health(container_per_test):
+    """Test the pytorch container."""
+
+    # chech pytorch Version
+    container_per_test.connection.check_output(
+        "python3.11 -c 'import torch; print(torch.__version__)'"
+    )
+    container_per_test.connection.check_output("git --version")
```

Comment:
No need to use the per_test variant here
```suggestion
    "container",
    [PYTORCH_CONTAINER],
    indirect=["container"],
)
def test_pytorch_health(container):
    """Test the pytorch container."""

    # chech pytorch Version
    container.connection.check_output(
        "python3.11 -c 'import torch; print(torch.__version__)'"
    )
    container.connection.check_output("git --version")
```